backends/kv/writer.go

```diff
@@ -26,11 +26,10 @@ import (
 	"time"
 
 	"github.com/nuclio/logger"
-	v3io "github.com/v3io/v3io-go/pkg/dataplane"
-
 	"github.com/v3io/frames"
 	"github.com/v3io/frames/backends/utils"
 	"github.com/v3io/frames/v3ioutils"
+	v3io "github.com/v3io/v3io-go/pkg/dataplane"
```

Comment:
Pretty sure it was in the right place before.

Reply:
the GoLand formatter did this 🤷‍♂

Reply:
Ok, not linter in Frames apparently.